files_airflow_ext/orchestrate/meltano.py

```diff
@@ -37,31 +37,69 @@
 
 if not Path(PROJECT_ROOT).joinpath(MELTANO_BIN).exists():
     logger.warning(
-        f"A symlink to the 'meltano' executable could not be found at '{MELTANO_BIN}'. Falling back on expecting it "
-        f"to be in the PATH instead. "
+        f"A symlink to the 'meltano' executable could not be found at '{MELTANO_BIN}'. "
+        "Falling back on expecting it to be in the PATH instead. "
     )
     MELTANO_BIN = "meltano"
 
 
+def _is_enabled(schedule_def: dict, environment_name: str) -> bool:
+    """Return true if the schedule should be enabled for the named environment."""
+    airbyte_annotations: dict = schedule_def.get("annotations", {}).get("airbyte", {})
```

Comment:
@aaronsteers I think these were supposed to be airflow references instead of airbyte 😄 , right?